src/App.svelte

```diff
@@ -121,678 +182,143 @@
-            ></p>
-            <div
-              class="hidden lg:inline-flex md:inline-flex flex flex-inline"
-            >
-              <div class="flex flex-inline">
-                <a target="_blank" href="https://github.com/AmandineFlachs">
-                  <i
-                    class="fa fa-github-alt pr-1 pt-6"
-                    style="
-                      font-size: 18px;
-                      color: rgb(109, 71, 199);
-                      font-style: normal;
-                    "
-                  ></i>
-                  <p
-                    class="inline pr-5 pt-5 text-purple-500 text-lg font-medium"
-                  >
-                    GitHub
-                  </p>
-                </a>
-              </div>
-              <div class="flex flex-inline">
-                <a target="_blank" href="https://www.flachsconsulting.com">
-                  <i
-                    class="fa fa-link pr-1 pt-6"
-                    style="
-                      font-size: 18px;
-                      color: rgb(109, 71, 199);
-                      font-style: normal;
-                    "
-                  ></i>
-                  <p
-                    class="inline pr-5 pt-5 text-purple-500 text-lg font-medium"
-                  >
-                    Website
-                  </p>
-                </a>
-              </div>
+            <div>
+              <a target="_blank" href={data.github}>
+                <i
+                  class="fa fa-github-alt pt-5 pr-5"
+                  style="font-size: 20px; color: rgb(109, 71, 199)"
+                ></i>
+              </a>
             </div>
-            <!-- for small screens -->
-            <div
-              class="flex-inline lg:hidden sm:flex-inline md:hidden flex flex-inline"
-            >
-              <div>
-                <a target="_blank" href="https://github.com/AmandineFlachs">
-                  <i
-                    class="fa fa-github-alt pt-5 pr-5"
-                    style="font-size: 20px; color: rgb(109, 71, 199)"
-                  ></i>
-                </a>
-              </div>
-              <div>
-                <a target="_blank" href="https://www.flachsconsulting.com">
-                  <i
-                    class="fa fa-link pt-5 pr-5"
-                    style="font-size: 20px; color: rgb(109, 71, 199)"
-                  ></i>
-                </a>
-              </div>
+            <div>
+              <a target="_blank" href={data.website}>
+                <i
+                  class="fa fa-link pt-5 pr-5"
+                  style="font-size: 20px; color: rgb(109, 71, 199)"
+                ></i>
+              </a>
             </div>
           </div>
         </div>
       </div>
     </div>
-  </div>
+  </div>  
+{/each}
+</div>
+    <!--Dont go below this line please !!!-->
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
are these spaces and comment necessary? Could you provide a little more information on why we can't go below the line?

Reply:
Na that's just for differentiating for anyone working on it .. since it's still a work in progress .. will do 

I have a bad habit of writing comments 😅

Reply:
you can create a draft PR which is the same as work in progress 😃, but no worries you're doing fine for your first PR 👍 👍